store/cachekv/store.go

```diff
@@ -332,9 +332,9 @@ func (store *Store) dirtyItems(start, end []byte) {
 	// if below minSortSize, expand it to cover additional values
 	// this amortizes the cost of processing elements across multiple calls
 	if endIndex-startIndex < minSortSize {
-		endIndex = math.MinInt(startIndex+minSortSize, len(strL)-1)
+		endIndex = math.Min(startIndex+minSortSize, len(strL)-1)
 		if endIndex-startIndex < minSortSize {
-			startIndex = math.MaxInt(endIndex-minSortSize, 0)
+			startIndex = math.Max(endIndex-minSortSize, 0)
 		}
```

Comment:
I don't think it's the right API. The build is failing.

Reply:
Oh may need a new release of math for the api.

Reply:
Ohh, I think you are talking about std math then. Our math packages does not have a `Min` and `Max`.
std math does: https://pkg.go.dev/math#Max

Reply:
Tagged a `cosmossdk.io/math v1.0.0-beta.5` 👍🏾

Reply:
thanks